src/index.js

```diff
@@ -4,21 +4,41 @@ import React from 'react';
 import { render } from 'react-dom';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import { CookiesProvider } from 'react-cookie';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider as ReduxProvider } from 'react-redux';
+import thunk from 'redux-thunk';
+import { createLogger } from 'redux-logger';
+import rootReducer from './reducers/index';
 
 import NotFound from '@/containers/Global/NotFound';
 
-import App from './containers/App';
+import RoverList from './containers/RoverList';
 import Accounts from './containers/Accounts/Base';
 
+const reduxLogger = createLogger();
+
+const store = createStore(
+  rootReducer,
+  applyMiddleware(
+    thunk,
+    reduxLogger,
+  ),
```

Comment:
Redux supports a middleware pattern, intercepting action dispatches as they are made and transforming them in a chain of function calls.  If you've ever worked with Express.js, you'll be familiar.  

We could use the [Promise middleware for redux](https://github.com/pburtchaell/redux-promise-middleware) to avoid a bit of boilerplate that you'll see in the actions and reducers files.